### PR TITLE
images: drop ABRT packages from fedora-rawhide

### LINKED ISSUE #6587: [no-test] Image refresh for fedora-rawhide
Image refresh for fedora-rawhide
 * [ ] FAIL: image-refresh fedora-rawhide

### PR DESCRIPTION
**Pull request by tomasmatus**
* [x] image-refresh fedora-rawhide

Fixes #6587

### CODE REVIEW

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -172,7 +164,7 @@ rpm -e --verbose man-db
 # no need to install all dependencies there
 ALL_DEPS="$COCKPIT_DEPS"
 if [ "fedora-eln" != "$IMAGE" ]; then
-    ALL_DEPS="$COCKPIT_MACHINES_DEPS $COCKPIT_STORAGE_DEPS $COCKPIT_DEPS $IPA_CLIENT_PACKAGES $TEST_PACKAGES $ABRT_PACKAGES"
+    ALL_DEPS="$COCKPIT_MACHINES_DEPS $COCKPIT_STORAGE_DEPS $COCKPIT_DEPS $IPA_CLIENT_PACKAGES $TEST_PACKAGES"
```
Comment (martinpitt) [inline images/scripts/fedora.setup:167]: This is too much. We still want to keep the ABRT_PACKAGES on fedora-{39,40}, just not on any newer release (i.e. rawhide).

Comment (tomasmatus) [reply images/scripts/fedora.setup]: OK, I think it should be better now

File: images/scripts/fedora.setup [hunk 2]
```diff
@@ -119,13 +119,15 @@ valkey \
...
 
-ABRT_PACKAGES="\
-abrt \
-abrt-addon-ccpp \
-reportd \
-libreport-plugin-bugzilla \
-libreport-fedora
-"
+if [ "$VERSION_ID" -le "40" ]; then
+    ABRT_PACKAGES="\
```
Comment (martinpitt) [inline images/scripts/fedora.setup:123]: This is a bit awkward and also get into trouble due to `set -u`. I'd recommend to leave this definition as it is, and instead conditionalize the `dnf install` command. There's no problem with splitting up the `dnf install` command into two.

Comment (tomasmatus) [reply images/scripts/fedora.setup]: ok, instead of splitting the `dnf install` command I added a condition where it adds ABRT packages to `ALL_DEPS`

**Comment by martinpitt** [comment]
Thanks! I set the bots in motion.

**Comment by cockpituous** [comment]
image-refresh fedora-rawhide done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-rawhide-20240708-084723

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-rawhide-9bb9b5b9-20240708-082118/log.html

**Review by martinpitt — APPROVED** [review]
Dekuji! Assuming green, this looks good.

**Pull request merged**